microservices-connector/cmd/router/main.go

```diff
@@ -45,6 +47,7 @@ const (
 	ServiceNode   = "node"
 	DataPrep      = "DataPrep"
 	Parameters    = "parameters"
+	UI            = "UI"
```

Comment:
> Hi @Ruoyu-y, I think this UI component is not the GMC UI, it's just for `ChatQnA` example, if so, how to re-name it to `ChatQnAUI` ?
This UI does not ONLY handle the case for ChatQnA, i also tried it with CodeGen. It can work too. From function perspective, i would think this is a UI config or handling logic for application serving. And i understand that the GMC UI you mentioned is for pipeline composition purpose, correct?